platforms/jvm/plugins-java-base/build.gradle.kts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("gradlebuild.distribution.api-java")
+}
+
+description = "Contains a basic JVM plugin used to compile, test, and assemble Java source; often applied by other JVM plugins (though named java-base, jvm-base would be a more proper name)."
+
+dependencies {
+    implementation(project(":base-services"))
+    implementation(project(":core-api"))
+    implementation(project(":core"))
+    implementation(project(":dependency-management"))
+    implementation(project(":diagnostics"))
+    implementation(project(":execution"))
+    implementation(project(":file-collections"))
+    implementation(project(":language-java"))
+    implementation(project(":language-jvm"))
+    implementation(project(":logging"))
+    implementation(project(":model-core"))
+    implementation(project(":platform-base"))
+    implementation(project(":platform-jvm"))
+    implementation(project(":plugins-jvm-test-suite-base"))
+    implementation(project(":reporting"))
+    implementation(project(":testing-base"))
+    implementation(project(":testing-jvm"))
+    implementation(project(":toolchains-jvm"))
+
+    implementation(libs.commonsLang)
+    implementation(libs.groovy)
+    implementation(libs.guava)
+    implementation(libs.inject)
+
+    testImplementation(testFixtures(project(":core")))
+
+    testRuntimeOnly(project(":distributions-jvm")) {
```

Comment:
Again it seems odd that our unit tests need to depend on the distribution. 

Is there any way we can avoid this? Can we just pull out the offending tests and move them in another PR?